repository.json

```diff
@@ -1509,14 +1509,50 @@
 		},
 		{
 			"name": "watchdog",
-			"description": "Python library to monitor filesystem events http://packages.python.org/watchdog/",
-			"author": "vovkkk",
-			"issues": "https://github.com/vovkkk/sublime-watchdog/issues",
+			"description": "Reusable constraint types to use with typing.Annotated.",
+			"author": "adriangb",
+			"issues": "https://github.com/annotated-types/annotated-types/issues",
```

Comment:
Those meta data don't seem to belong to watchdog.

Reply:
oops some copy paste mistake :(